tests/unit/plugins/module_utils/test_interfaces_assignments_utils.py

```diff
@@ -0,0 +1,1533 @@
+                    <uid>0</uid>
+                    </user>
+                    <user>
+                    <password>$2y$10$1BvUdvwM.a.dJACwfeNfAOgNT6Cqc4cKZ2F6byyvY8hIK9I8fn36O</password>
+                    <scope>user</scope>
+                    <name>vagrant</name>
+                    <descr>vagrant box management</descr>
+                    <expires/>
+                    <authorizedkeys/>
+                    <ipsecpsk/>
+                    <otp_seed/>
+                    <shell>/bin/sh</shell>
+                    <uid>1000</uid>
+                    </user>
+                    <nextuid>2001</nextuid>
+                    <nextgid>2000</nextgid>
+                    <timezone>Etc/UTC</timezone>
+                    <time-update-interval>300</time-update-interval>
+                    <timeservers>0.nl.pool.ntp.org</timeservers>
+                    <webgui>
+                    <protocol>https</protocol>
+                    <ssl-certref>5a3951eaa0f49</ssl-certref>
+                    </webgui>
+                    <disablenatreflection>yes</disablenatreflection>
+                    <usevirtualterminal>1</usevirtualterminal>
+                    <disableconsolemenu/>
+                    <disablechecksumoffloading>1</disablechecksumoffloading>
+                    <disablesegmentationoffloading>1</disablesegmentationoffloading>
+                    <disablelargereceiveoffloading>1</disablelargereceiveoffloading>
+                    <ipv6allow/>
+                    <powerd_ac_mode>hadp</powerd_ac_mode>
+                    <powerd_battery_mode>hadp</powerd_battery_mode>
+                    <powerd_normal_mode>hadp</powerd_normal_mode>
+                    <bogons>
+                    <interval>monthly</interval>
+                    </bogons>
+                    <kill_states/>
+                    <enablesshd/>
+                    <ssh>
+                    <enabled>enabled</enabled>
+                    <permitrootlogin>1</permitrootlogin>
+                    <passwordauth>0</passwordauth>
+                    </ssh>
+                    <backupcount>60</backupcount>
+                    <crypto_hardware>aesni</crypto_hardware>
+                    <backup>
+                    <nextcloud version="1.0.0">
+                    <enabled>0</enabled>
+                    <url/>
+                    <user/>
+                    <password/>
+                    <password_encryption/>
+                    <backupdir>OPNsense-Backup</backupdir>
+                    </nextcloud>
+                    </backup>
+                    <firmware version="1.0.1">
+                    <mirror/>
+                    <flavour/>
+                    <plugins/>
+                    <type/>
+                    <subscription/>
+                    <reboot/>
+                    </firmware>
+                </system>
+                <interfaces>
```

Comment:
Is it possible to check in just L289-L372 for those tests? I think that might be enough for these tests

Reply:
For the sake of integrity, I would like to have a full config to test the code.